docs/02-app/01-building-your-application/07-configuring/03-environment-variables.mdx

```diff
@@ -183,6 +183,8 @@ If you've configured [Development Environment Variables](https://vercel.com/docs
 vercel env pull .env.local
 ```
 
+> **Good to know**: When deploying your Next.js application to [Vercel](https://vercel.com), your environmet variables in `.env*` files will not be made available to Edge Runtime, unless their name are prefided with `NEXT_PUBLIC_`. We strognly recommend managing your environment variables in [Project Settings](https://vercel.com/docs/projects/environment-variables?utm_medium=docs&utm_source=next-site&utm_campaign=next-website) instead.
```

Comment:
**Good to know**: When deploying your Next.js application to [Vercel](https://vercel.com), your environment variables in `.env*` files will not be made available to Edge Runtime, unless their name are prefixed with `NEXT_PUBLIC_`. We strongly recommend managing your environment variables in [Project Settings](https://vercel.com/docs/projects/environment-variables?utm_medium=docs&utm_source=next-site&utm_campaign=next-website) instead.

Reply:
Just corrected spelling :)

Reply:
omg lol thanks for catching my typos!